src/resources/container.js

```diff
@@ -0,0 +1,157 @@
+Object.assign(pc, function () {
+
+    /**
+     * @class
+     * @name pc.ContainerResource
+     * @classdesc Container for a list of animations, textures and a model.
+     * @param {Object} data - The loaded GLB data.
+     */
+    var ContainerResource = function (data) {
```

Comment:
Feels to me that since this is only for GLB stuff we should name it GlbResource. Also we should initialize all the properties of the class in the constructor so that we know what it contains instead of initializing fields later in the resource handler. I believe there are also performance implications if fields are set without first having been created in the constructor.

Reply:
Yep agreed, thanks.

Reply:
Without understanding all the details, I like the generic ContainerResource name. It opens the doors to have multiple resources in a single asset, which might be very useful once we allow some Editor plugins to modify import pipeline. I used this few times with other engines, where your custom import generates multiple different assets from a single source asset.